boards/96boards/wistrio/96b_wistrio.yaml

```diff
@@ -12,3 +12,4 @@ supported:
   - i2c
   - eeprom
   - lora
+  - lis2dh
```

Comment:
Instead of the specific sensor driver implementation, I think this should be the sensor class. e.g., `accel`.

Reply:
and how the pairing will work? for example `samples/sensor/lis2dh/` in this case will be built on all boards with `accel` even if it is a different sensor? That is not how most board do this right now, we point to the actual implementation.

Reply:
We need to remove the remaining driver-specific samples. The pairing should reflect interfaces, like for other driver classes. I2C for example, we say is supported as a driver class or interface, not as a specific implementation.

Reply:
but that does not reflect the state of things in the tree right now, sensors are added as the implementation and not the interface, see `boards/st/disco_l475_iot1/disco_l475_iot1.yaml`

```
supported:
  - arduino_gpio
  - arduino_i2c
  - i2c
  - hts221
  - lps22hb
  - lsm6dsl
  - pwm
  - counter
  - gpio
  - ble
  - spi
  - nvs
  - vl53l0x
  - watchdog
  - adc
  - dac
  - qspi
  - dma
  - rtc
  - usbd
  ```

We can improve this and do something like we do with network, i.e. 

`sensor:accel:lis2dh` or something along those lines, but this PR changes based on what we have right now, such an enhancement can be done as a followup.